com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1182,6 +1180,32 @@ def __init__(self, rest_client, credentials):
     def _get_url(self, req_name):
         return self._credentials['rest_url'] + self._credentials[req_name]
 
+    def submit_job(self, sab_file, configuration=None):
+        """Submit a compiled Streams Application Bundle (a SAB file) to run in this Streamin Analytics service, optionally with a job configuration overlay.
+        
+        Args:
+        sab_file(str): path to a compiled SAB file containing the application to be submitted
+        configuration(dict): job configuration overlay containing application parameters, deployment options, tracing level, etc.
```

Comment:
Isn't this a `JobConfig` not a `dict`?

Reply:
In the other PR I said there were two options we wanted to support:

- a JCO file/object
- a `JobConfig` object

If this just supports a JCO object then it's ok - others can be added later.